charts/consul/Chart.yaml

```diff
@@ -10,14 +10,14 @@ sources:
   - https://github.com/hashicorp/consul
   - https://github.com/hashicorp/consul-k8s
 annotations:
-  artifacthub.io/prerelease: false
+  artifacthub.io/prerelease: true
   artifacthub.io/images: |
     - name: consul
-      image: hashicorp/consul:1.15.5
+      image: docker.mirror.hashicorp.services/hashicorppreview/consul:1.15-dev
     - name: consul-k8s-control-plane
-      image: hashicorp/consul-k8s-control-plane:1.1.5
+      image: docker.mirror.hashicorp.services/hashicorppreview/consul-k8s-control-plane:1.1.6-dev
     - name: consul-dataplane
-      image: hashicorp/consul-dataplane:1.1.5
+      image: docker.mirror.hashicorp.services/hashicorppreview/consul-dataplane:1.1-dev
```

Comment:
Does this need to be 1.1.6?

Reply:
https://hub.docker.com/layers/hashicorppreview/consul-dataplane/1.1-dev/images/sha256-c137bf627dd8cb4b71a33f475d773b4cc2026692f13bac267b876115cde6f04b?context=explore

This looks like the dev tag for dataplane 1.1.x to me. Its autogenerated by the make prepare-dev command